docs/spec.md

```diff
@@ -1724,27 +1731,30 @@ when `comparison_direction` is `GE`, `GT`, `LE` or `LT`
 
 | Label | Name                   | Type                                                    | Constraints |
 |-------|------------------------|---------------------------------------------------------|-------------|
-| (I1)  | `lhs`                  | tensor                                                  | (C1-C3)     |
-| (I2)  | `rhs`                  | tensor                                                  | (C1), (C2)  |
+| (I1)  | `lhs`                  | tensor or per-tensor quantized tensor                   | (C1-C4)     |
+| (I2)  | `rhs`                  | tensor or per-tensor quantized tensor                   | (C1-C2)     |
 | (I3)  | `comparison_direction` | enum of `EQ`, `NE`, `GE`, `GT`, `LE`, and `LT`          |             |
 | (I4)  | `compare_type`         | enum of `FLOAT`, `TOTALORDER`, `SIGNED`, and `UNSIGNED` | (C3)        |
 
 #### Outputs
 
-| Name     | Type                   | Constraints |
-|----------|------------------------|-------------|
-| `result` | tensor of boolean type | (C2)        |
+| Name     | Type                                                | Constraints |
+|----------|-----------------------------------------------------|-------------|
+| `result` | tensor of boolean type or per-tensor quantized type | (C2)        |
```

Comment:
I think we should keep just "tensor of boolean type" here. 

This follows the spirit of "if a thing doesn't support a tensor of floating-point type, then it shouldn't support a quantized tensor", which led us to decline support for quantization in integer-only operations. Similarly, in IsFiniteOp, we don't say that its output can be a quantized tensor, we just keep "tensor of boolean type".